RadarSDK/Radar.h

```diff
@@ -7,6 +7,7 @@
 
 #import <CoreLocation/CoreLocation.h>
 #import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
```

Comment:
Intentional?

Reply:
I mean should we put it in the .h or .m file

Reply:
Good point, added to listen for `UIApplicationWillEnterForegroundNotification` for `sessionId`, but can probably move it to the `.m` file